package.json

```diff
@@ -13,7 +13,8 @@
     "strapi": "strapi",
     "test": "playwright test",
     "test:postgres": "TEST_CONTAINER=true nyc playwright test",
-    "test:coverage": "nyc playwright test"
+    "test:coverage": "nyc playwright test",
+     "install-all": "npm install --prefix /home/runner/work/strapiv4/strapiv4/"
```

Comment:
Where are we calling `install-all`? Note that https://github.com/dev-launchers/strapiv4/blob/main/.github/workflows/publish-types.yaml#L26 still goes into the type folder. Do we need to go into the types folder?

Reply:
I removed install-all and it didn't change notting inside the code working.
We don't need to go to type folder for running our npm publish
We can do it by '...your path/strapiv4' or '...your path/strapiv4/types'